### PR TITLE
Fix compilation errors with 5.5 preview

### PR DESCRIPTION
**Pull request by Murazaki**
Fix two deprecated elements, a function and enum.

### CODE REVIEW

File: Source/DlgSystemEditor/Browser/SDlgBrowser.cpp [hunk 1]
```diff
@@ -1097,7 +1097,7 @@ TSharedRef<SWidget> SDlgBrowser::MakeButtonWidgetForGraphNodes(
 )
 {
 	TSharedPtr<SWrapBox> Buttons = SNew(SWrapBox)
-			.PreferredWidth(600.f);
+			.PreferredSize(600.f);
```
[the remark this answers is not in the document]
Comment (Murazaki) [reply Source/DlgSystemEditor/Browser/SDlgBrowser.cpp:1100]: Don't have the complete details on that but I can wrap that in preprocessors if you prefer

[the remark this answers is not in the document]
Comment (Murazaki) [reply Source/DlgSystemEditor/Browser/SDlgBrowser.cpp:1100]: According to 5.3.0, it seems like it
https://github.com/EpicGames/UnrealEngine/blob/40eea367040d50aadd9f030ed5909fc890c159c2/Engine/Source/Runtime/Slate/Public/Widgets/Layout/SWrapBox.h#L155C1-L161C1

**Comment by Murazaki** [comment]
Is there more reasons for not accepting the PR ?

**Review by vampy — APPROVED** [review]

**Comment by vampy** [comment]
Sorry, forgot about this.

**Pull request merged**